packages/react-components/src/AccessibilityScenarios/Input.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Input } from '@fluentui/react-input';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Label } from '@fluentui/react-label';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button } from '@fluentui/react-button';
+
+import { Scenario } from './utils';
+
+export const RegistrationFormInputsAccessibilityScenario: React.FunctionComponent = () => {
+  return (
+    <Scenario pageTitle="Registration form inputs">
+      <h1>Registration form</h1>
+
+      <Label htmlFor="name">Name:</Label>
```

Comment:
Since this is a form example (complete with submit button) it probably makes sense to wrap it in `<form>`

Reply:
Done.